protos/flyteidl/core/tasks.proto

```diff
@@ -238,3 +239,22 @@ message DataLoadingConfig {
     LiteralMapFormat format = 4;
     IOStrategy io_strategy = 5;
 }
+
+// Defines a pod spec and additional pod metadata that is created when a task is executed.
+message K8sPod {
+   // Contains additional metadata for building a kubernetes pod.
+   K8sObjectMetadata metadata = 1;
+
+   // Defines the primary pod spec created when a task is executed.
```

Comment:
nit: want to add a link to the expected format here? maybe the python and go types?

Reply:
sure, done

Reply:
So we need to add K8sObject prefix? Can we just make it metadata. Not sure about the namespacing

Reply:
i think it's package namespaced, right? even in the context of this file i feel like metadata could be ambiguous